pkg/registry/interface.go

```diff
@@ -10,7 +10,7 @@ type Load interface {
 	AddOperatorBundle(bundle *Bundle) error
 	AddBundleSemver(graph *Package, bundle *Bundle) error
 	AddPackageChannels(manifest PackageManifest) error
-	AddBundlePackageChannels(manifest PackageManifest, bundle *Bundle) error
+	// AddBundlePackageChannels(manifest PackageManifest, bundle *Bundle) error
```

Comment:
Note: commented this out because it doesn't seem to be used anywhere

Reply:
could we remove instead, so?

Reply:
sounds good - will do! Just thought to flag it here for visibility